### PR TITLE
Document changes to sysid and component id

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates docs with changes resulting from https://github.com/Dronecode/camera-streaming-daemon/pull/166

### CODE REVIEW

File: en/guide/overview.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ CSD supports the following key features:
 * Automatically attaches [compatible cameras](#supported_cameras) connected to the Linux computer when it is started.
 * RTSP video streaming from *all* connected cameras (for consumption by GCS or other video players).
 * RTSP video stream advertising/discovery using Avahi.
-* [MAVLink Camera Protocol](#mavlink_support) support for up to 5 cameras, enabling image/video capture and storage, and querying/setting camera options.
+* [MAVLink Camera Protocol](#mavlink_support) support for up to 6 cameras, enabling image/video capture and storage, and querying/setting camera options.
```
Comment (lbegani) [inline en/guide/overview.md]: Should image capture, video capture be part of key feature of CM?

File: en/guide/overview.md [hunk 2]
```diff
@@ -44,16 +44,18 @@ Advanced configuration information about individual cameras is specified in [Cam
...
 
-Component IDs for each camera are allocated automatically and sequentially from [MAV_COMP_ID_CAMERA2](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA2) to [MAV_COMP_ID_CAMERA6](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA6) (inclusive) as cameras are connected (once all component ids are allocated further cameras are not addressable).
+All cameras share the same MAVLink system ID. This can (optionally) be defined in the configuration file to match the id of the associated autopilot. If the system id is not defined in the configuration file then CSD will initially use the default PX4 vehicle system ID (1) and then change to the id of the first autopilot it detects.
+
+Component IDs for each camera are allocated automatically and sequentially from [MAV_COMP_ID_CAMERA](https://mavlink.io/en/messages/common.html#MAV_COMP_ID_CAMERA) as cameras are connected (once all component ids are allocated further cameras are not addressable). For Gazebo-enabled CSD builds the first ID is allocated to the Gazebo plugin camera.
```
Comment (lbegani) [inline en/guide/overview.md]: > For Gazebo-enabled CSD builds the first ID is allocated to the Gazebo plugin camera
Not sure, what it means

Comment (hamishwillee) [reply en/guide/overview.md]: Thanks. I re-wrote as "When the CSD is built with Gazebo enabled, the Gazebo camera has component ID `MAV_COMP_ID_CAMERA`."

File: en/guide/overview.md (hunk 1 above)
Comment (hamishwillee) [reply en/guide/overview.md]: Yes. They are, but they are part of "MAVLink support". That is why the doc says "... support for up to 6 cameras, enabling image/video capture and storage, and querying/setting camera options."

**Comment by hamishwillee** [comment]
Thanks @lbegani . I have fixed the main issue and merged. If you think not well enough done, let me know and we can fix in a separate PR.

**Pull request merged**